admin_guide/out_of_resource_handling.adoc

```diff
@@ -251,79 +285,41 @@ of service for the pod.
 |===
 
 If the node is unable to reclaim memory prior to experiencing a system OOM
-event, the `*oom_killer*` calculates an `*oom_score*` based on the percentage of
-memory a container is using on the node, and then add the `*oom_score_adj*` to get
-an effective `*oom_score*` for the container, and then kills the container with
-the highest score.
+event, the `*oom_killer*` calculates an `*oom_score*`:
+
+----
+% of node memory a container is using + `*oom_score_adj*` = `*oom_score*`
+----
 
-The intended behavior is that containers with the lowest quality of service that
-are consuming the largest amount of memory relative to the scheduling request
-are killed first in order to reclaim memory.
+The node then kills the container with the highest score.
 
-Unlike pod eviction, if a pod container is OOM killed, it may be restarted by
+Containers with the lowest quality of service that are consuming the largest
+amount of memory relative to the scheduling request are killed first.
+
+Unlike pod eviction, if a pod container is OOM killed, it can be restarted by
 the node based on its `*RestartPolicy*`.
 
-[[out-of-resource-best-practices]]
-== Best Practices
+[[out-of-resource-recommended-practices]]
+== Recommended Practices
 
 [[out-of-resource-best-practice-disable-swap-memory]]
 === Disable Swap Memory
 
-Failure to disable swap memory makes the node not recognize it is under
+Failure to disable swap memory makes the node not recognize it under
```

Comment:
s/it/it's

I think this is so important that it should be called out maybe in the top of the document with something like ensuring your node has been configured correctly.